### ISSUE TITLE
SMART Authentication?

### ISSUE THREAD

**Issue by RoyalJoie** [p2/growth-charts#29]
Does the app allow a patient to automatically connect to their health records using their existing patient portal credentials? Or is it a manual data entry?

**Comment by p2** [comment]
No, unfortunately the official version of the app does not support SMART authentication.
